cruise/CruiseInformation.csv

```diff
@@ -200,7 +237,8 @@ EK20190515,R/V Elakha,2019-05-15T15:30:00,2019-05-15T22:00:00,Endurance glider d
 TS20190516,F/V Tenacious II,2019-05-16T12:00:00,2019-05-17T12:00:00,Pioneer 12a glider deployment
 AR35-03,R/V Neil Armstrong,2019-05-27T00:00:00,2019-06-20T23:59:59,Irminger Glider gi_363 Recovery - Niskine Process Cruise
 EK20190529,R/V Elakha,2019-05-29T16:22:00,2019-05-29T19:33:00,Endurance ce01issp_00013 deployment
-AT-42-12,R/V Atlantis,2019-06-02T00:00:00,2019-07-09T00:00:00,Cabled (rvdata) Visions 19
+AT-42-12,R/V Atlantis,2019-06-02T00:00:00,2019-07-09T00:00:00,Cabled (rvdata) Visions 19 #TO BE DELETED
+AT42-12,R/V Atlantis,2019-06-02T00:00:00,2019-07-09T00:00:00,Cabled (rvdata) Visions 19
 ML20190612,R/V Miss Linda,2019-06-12T13:00:00,2019-06-13T18:00:00,Endurance glider recovery
 WJ20190617,M/V Warren Jr.,2019-06-17T18:00:00,2019-06-20T00:04:00,Pioneer 12b glider and AUV deployment
 EK20190624,R/V Elakha,2019-06-24T14:55:00,2019-06-24T19:00:00,Endurance ce02shsp_00013 deployment

```

Comment:
MZ20190626 should be here to be chronological.

Reply:
done

Reply:
looks good